utils/horribleHelper.js

```diff
@@ -7,9 +7,21 @@ const horribleHelper = {};
 
 const { HORRIBLE_URI } = process.env
 
-horribleHelper.magnet = async function(args, message) {
-  await axios
-    .get(`${HORRIBLE_URI}/${args}`)
+horribleHelper.magnet = function(args, message) {
+  const argsArray = args.split('/')
+  const anime = argsArray[0].trim()
+
+  if (!anime) {
+    message.channel.send(
+      enhanceChat.embedStatic('Try **!magnet naruto** or **magnet 323**', 'Missing anime name or id', '#bf0000')
+    )
+    return
+  }
+  const episode = argsArray[1] && argsArray[1].trim()
+  const url = parseInt(episode, 10) > 0 ? `${HORRIBLE_URI}/?anime=${anime}&ep=${episode}` : `${HORRIBLE_URI}/?anime=${anime}`
```

Comment:
better than this is to make params and pass that in `axios.get`
Something like:
```js
const params = {
    anime,
    ep: parseInt(episode, 10) > 0 ? episode : undefined
}
const { data } = await axios.get(HORRIBLE_URI, { params });
message.channel.send(enhanceChat.embedMagnets(data));
```